iocore/net/SSLNetVConnection.cc

```diff
@@ -980,6 +980,14 @@ SSLNetVConnection::sslStartHandShake(int event, int &err)
       } else {
         clientCTX = params->client_ctx;
       }
+
+      if (this->options.clientVerificationFlag && params->clientCACertFilename != nullptr && params->clientCACertPath != nullptr) {
```

Comment:
Nit pick, but for future, I'd much prefer e.g.

```
this->options.clientVerificationFlag && (params->clientCACertFilename != nullptr)
```

it just looks better, and feels a lot safer. Yes, I know they are the same.